### PR TITLE
chore: consolidate classic-level usage

### LINKED ISSUE #6794: Replace `leveldown` with `classic-level` in db tests
this PR (and this old dependency version) highlights that we're using a now-deprecated `leveldown` in our db tests. It should be replaced with `classic-level`.

_Originally posted by @wemeetagain in https://github.com/ChainSafe/lodestar/pull/6793#discussion_r1603298992_

### PR DESCRIPTION
**Pull request by jeluard**
**Motivation**

Make sure `classic-level` is used consistently.

**Description**

Remove older dependencies `level` and `leveldown`. `db` is already dependent on `classic-level`.

Closes #6794

### CODE REVIEW

File: packages/db/package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "@chainsafe/ssz": "^0.15.1",
     "@lodestar/config": "^1.18.1",
     "@lodestar/utils": "^1.18.1",
+    "classic-level": "^1.4.1",
```
Comment (jeluard) [inline packages/db/package.json:41]: It might make sense to depend on `classic-level` dependency directly now that we are requiring it.

Comment (wemeetagain) [reply packages/db/package.json:41]: Rather than using the `level` package?

Comment (jeluard) [reply packages/db/package.json:41]: Right, the code already expect running under NodeJS and we add `classic-level` dependency to make sure the right version is used.

Comment (wemeetagain) [reply packages/db/package.json:41]: Sounds good to me.
Can you update this PR to include that? Currently it feels like in a halfway state (which I suppose caused your comment here).

Comment (jeluard) [reply packages/db/package.json:41]: @wemeetagain done

**Review by wemeetagain — APPROVED** [review]
lgtm thanks

**Comment by wemeetagain** [comment]
failing CI

**Comment by wemeetagain** [comment]
:tada: This PR is included in [v1.19.0](https://github.com/ChainSafe/lodestar/releases/tag/v1.19.0) :tada:

**Pull request merged**